port/linuxppc64/omrtime.c

```diff
@@ -165,14 +165,18 @@ omrtime_nano_time(struct OMRPortLibrary *portLibrary)
 uint64_t
 omrtime_hires_clock(struct OMRPortLibrary *portLibrary)
 {
-	struct timeval tp;
+	uint64_t ret;
+	struct timespec ts;
 
 	if (systemcfgP_nanos) {
 		return __getNanos();
 	}
 
-	gettimeofday(&tp, NULL);
-	return ((int64_t)tp.tv_sec) * OMRTIME_HIRES_CLOCK_FREQUENCY + tp.tv_usec * (OMRTIME_HIRES_CLOCK_FREQUENCY / 1000000);
```

Comment:
Rewrote to meet coding standards:
- `ret` should be initialized when declared;
- single return statement is preferred; and
- scope of the timespec variable is reduced via the else statement.
```
	uint64_t ret = 0;
	if (systemcfgP_nanos) {
		ret = __getNanos();
	} else {
		struct timespec ts;
		if (0 == clock_gettime(CLOCK_MONOTONIC_RAW, &ts)) {
			ret = ((uint64_t)ts.tv_sec * OMRTIME_NANOSECONDS_PER_SECOND) + (uint64_t)ts.tv_nsec;
		}
	}
	return ret;

Reply:
I updated the code as suggested.